recipes/rebound/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "rebound" %}
+{% set version = "3.4.0" %}
+{% set sha256 = "d070bfd7a8c48a28776b4bae5ba4dcf22d81e7776cd518dc3a3479972104d5bc" %}
+
+package:
+  name: {{ name|lower }}
+  version:  {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url:  https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+
+build:
+  skip: true  # [win]
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  imports:
+    - rebound
+
+
+
+about:
+  home: http://github.com/hannorein/rebound
+  license: GPL3
+  license_family: GPL
+  summary: 'An open-source multi-purpose N-body code'
+
+
+  description:  |
+    REBOUND is an N-body integrator, i.e. a software package that can
+    integrate the motion of particles under the influence of gravity.
+  doc_url: http://rebound.readthedocs.io/en/latest/
+  dev_url: https://github.com/hannorein
+
+extra:
+  recipe-maintainers:
+    -ACBlock
+    -mwcraig
+    -hannorein
```

Comment:
Added PR ( https://github.com/conda-forge/staged-recipes/pull/3048 ) to fix this.